src/System.Private.CoreLib/src/System/Exception.cs

```diff
@@ -337,7 +337,7 @@ public virtual string Source
             {
                 if (_source == null)
                 {
-                    StackTrace st = new StackTrace(this, true);
+                    StackTrace st = new StackTrace(this, false);
```

Comment:
Nit it is nice for the reader to name param in the case of a boolean literal ie
`new StackTrace(this, fNeedFileInfo: false)`

Some intersting Hungarian used there.